Unset -Werror in the bcmdhd* drivers, because they cause the build to fail with GCC6.

--- a/drivers/net/wireless/bcmdhd_4330/Makefile
+++ b/drivers/net/wireless/bcmdhd_4330/Makefile
@@ -5,7 +5,7 @@
         -DUNRELEASEDCHIP -Dlinux -DDHD_SDALIGN=64 -DMAX_HDR_READ=64           \
         -DDHD_FIRSTREAD=64 -DDHD_GPL -DDHD_SCHED -DBDC -DTOE -DDHD_BCMEVENTS  \
         -DSHOW_EVENTS -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS      \
-        -Wall -Wstrict-prototypes -Werror  -DSDIO_ISR_THREAD 		      \
+        -Wall -Wstrict-prototypes -DSDIO_ISR_THREAD 		      \
         -DEMBEDDED_PLATFORM -DARP_OFFLOAD_SUPPORT -DPKT_FILTER_SUPPORT        \
         -DKEEP_ALIVE -DCONFIG_FIRST_SCAN -DAP_ONLY -DCUSTOM_OOB_GPIO_NUM=299  \
         -DOOB_INTR_ONLY -DMMC_SDIO_ABORT                                      \
@@ -21,7 +21,7 @@ DHDCFLAGS = -DLINUX -DBCMDRIVER -DBCMDONGLEHOST -DUNRELEASEDCHIP -DBCMDMA32  \
 	-DCUSTOM_OOB_GPIO_NUM=46 -DOOB_INTR_ONLY -DMMC_SDIO_ABORT -DEMBEDDED_PLATFORM -DCUSTOMER_HW2 -DDHD_PRINT_DEBUG   \
 	-DPNO_SUPPORT -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS -DWIFI_ACT_FRAME -DKEEP_ALIVE -DDHD_DEBUG -DWL_ENABLE_P2P_IF -DHAVE_FREE_NETDEV\
 	-DDHD_USE_IDLECOUNT -DCSCAN -DHTC_KlocWork -DWL_CFG80211 -DWLP2P -DWL_COMPAT_WIRELESS -DENABLE_INSMOD_NO_FW_LOAD -DBCMWAPI_WAI -DBCMWAPI_WPI \
-	-Wall -Wstrict-prototypes -Werror                                    \
+	-Wall -Wstrict-prototypes                                     \
 	-Idrivers/net/wireless/bcmdhd_4330 -Idrivers/net/wireless/bcmdhd_4330/include
 	#-I$(M) -I$(M)/include
 	# CFG 80211 and P2P compile flag
--- a/drivers/net/wireless/bcmdhd_4334/Makefile
+++ b/drivers/net/wireless/bcmdhd_4334/Makefile
@@ -5,7 +5,7 @@
         -DUNRELEASEDCHIP -Dlinux -DDHD_SDALIGN=64 -DMAX_HDR_READ=64           \
         -DDHD_FIRSTREAD=64 -DDHD_GPL -DDHD_SCHED -DBDC -DTOE -DDHD_BCMEVENTS  \
         -DSHOW_EVENTS -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS      \
-        -Wall -Wstrict-prototypes -Werror  -DSDIO_ISR_THREAD 		      \
+        -Wall -Wstrict-prototypes -DSDIO_ISR_THREAD 		      \
         -DEMBEDDED_PLATFORM -DARP_OFFLOAD_SUPPORT -DPKT_FILTER_SUPPORT        \
         -DKEEP_ALIVE -DCONFIG_FIRST_SCAN -DAP_ONLY -DCUSTOM_OOB_GPIO_NUM=299  \
         -DOOB_INTR_ONLY -DMMC_SDIO_ABORT                                      \
@@ -22,7 +22,7 @@ DHDCFLAGS = -DLINUX -DBCMDRIVER -DBCMDONGLEHOST -DUNRELEASEDCHIP -DBCMDMA32  \
 	-DPNO_SUPPORT -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS -DWIFI_ACT_FRAME -DKEEP_ALIVE -DDHD_DEBUG -DWL_ENABLE_P2P_IF\
 	-DDHD_USE_IDLECOUNT -DCSCAN -DHTC_KlocWork -DWL_CFG80211 -DWLP2P -DWL_COMPAT_WIRELESS -DENABLE_INSMOD_NO_FW_LOAD -DBCMWAPI_WAI -DBCMWAPI_WPI \
 	-DDHD_USE_STATIC_BUF -DOLD_CFG_80211 -DHAVE_FREE_NETDEV\
-	-Wall -Wstrict-prototypes -Werror                                    \
+	-Wall -Wstrict-prototypes                                    \
 	-Idrivers/net/wireless/bcmdhd_4334 -Idrivers/net/wireless/bcmdhd_4334/include
 	#-I$(M) -I$(M)/include
 	# CFG 80211 and P2P compile flag
--- a/drivers/net/wireless/bcmdhd_4334_mcc/Makefile
+++ b/drivers/net/wireless/bcmdhd_4334_mcc/Makefile
@@ -2,7 +2,7 @@
 #####################
 # SDIO Basic feature
 #####################
-DHDCFLAGS = -Wall -Wstrict-prototypes -Werror -DLINUX -Dlinux        \
+DHDCFLAGS = -Wall -Wstrict-prototypes -DLINUX -Dlinux        \
             -DBCMDRIVER -DBCMDONGLEHOST -DUNRELEASEDCHIP -DBCMDMA32  \
             -DDHD_FIRSTREAD=128 -DMAX_HDR_READ=128 -DBCMFILEIMAGE    \
             -DDHDTHREAD -DBDC -DDHD_BCMEVENTS -DSHOW_EVENTS          \
--- a/drivers/net/wireless/bcmdhd_4335/Makefile
+++ b/drivers/net/wireless/bcmdhd_4335/Makefile
@@ -5,7 +5,7 @@
         -DUNRELEASEDCHIP -Dlinux -DDHD_SDALIGN=64 -DMAX_HDR_READ=64           \
         -DDHD_FIRSTREAD=64 -DDHD_GPL -DDHD_SCHED -DBDC -DTOE -DDHD_BCMEVENTS  \
         -DSHOW_EVENTS -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS      \
-        -Wall -Wstrict-prototypes -Werror  -DSDIO_ISR_THREAD 		      \
+        -Wall -Wstrict-prototypes -DSDIO_ISR_THREAD 		      \
         -DEMBEDDED_PLATFORM -DARP_OFFLOAD_SUPPORT -DPKT_FILTER_SUPPORT        \
         -DKEEP_ALIVE -DCONFIG_FIRST_SCAN -DAP_ONLY -DCUSTOM_OOB_GPIO_NUM=299  \
         -DOOB_INTR_ONLY -DMMC_SDIO_ABORT                                      \
@@ -24,7 +24,7 @@ DHDCFLAGS = -DLINUX -DBCMDRIVER -DBCMDONGLEHOST -DUNRELEASEDCHIP -DBCMDMA32  \
 	-DDHD_USE_STATIC_BUF -DOLD_CFG_80211 -DHAVE_FREE_NETDEV \
 	-DSUPPORT_MULTIPLE_REVISION -DENABLE_BCN_LI_BCN_WAKEUP -DWL11U -DVSDB -DHT40_GO -DWL_CFG80211_VSDB_PRIORITIZE_SCAN_REQUEST -DSUPPORT_MULTIPLE_REVISION -DUSE_CID_CHECK -DCONFIG_CONTROL_PM \
 	-DWLLXIW -DUSE_IW \
-	-Wall -Wstrict-prototypes -Werror                                    \
+	-Wall -Wstrict-prototypes                                    \
 	-I$(M) -I$(M)/include \
 	-Idrivers/net/wireless/bcmdhd_4335 -Idrivers/net/wireless/bcmdhd_4335/include
 	# CFG 80211 and P2P compile flag
--- a/drivers/net/wireless/bcmdhd_4335_mcc/Makefile
+++ b/drivers/net/wireless/bcmdhd_4335_mcc/Makefile
@@ -5,7 +5,7 @@
         -DUNRELEASEDCHIP -Dlinux -DDHD_SDALIGN=64 -DMAX_HDR_READ=64           \
         -DDHD_FIRSTREAD=64 -DDHD_GPL -DDHD_SCHED -DBDC -DTOE -DDHD_BCMEVENTS  \
         -DSHOW_EVENTS -DBCMSDIO -DDHD_GPL -DBCMLXSDMMC -DBCMPLATFORM_BUS      \
-        -Wall -Wstrict-prototypes -Werror  -DSDIO_ISR_THREAD 		      \
+        -Wall -Wstrict-prototypes -DSDIO_ISR_THREAD 		      \
         -DEMBEDDED_PLATFORM -DPKT_FILTER_SUPPORT        \
         -DKEEP_ALIVE -DCONFIG_FIRST_SCAN -DAP_ONLY -DCUSTOM_OOB_GPIO_NUM=299  \
         -DOOB_INTR_ONLY -DMMC_SDIO_ABORT                                      \
@@ -25,7 +25,7 @@ DHDCFLAGS = -DLINUX -DBCMDRIVER -DBCMDONGLEHOST -DUNRELEASEDCHIP -DBCMDMA32  \
 	-DSUPPORT_MULTIPLE_REVISION -DENABLE_BCN_LI_BCN_WAKEUP -DWL11U -DVSDB -DHT40_GO -DWLTDLS -DWL_CFG80211_VSDB_PRIORITIZE_SCAN_REQUEST -DSUPPORT_MULTIPLE_REVISION -DUSE_CID_CHECK -DCONFIG_CONTROL_PM \
         -DWL_SUPPORT_BACKPORTED_KPATCHES \
 	-DWLLXIW -DUSE_IW \
-	-Wall -Wstrict-prototypes -Werror                                    \
+	-Wall -Wstrict-prototypes                                    \
 	-I$(M) -I$(M)/include \
 	-Idrivers/net/wireless/bcmdhd_4335_mcc -Idrivers/net/wireless/bcmdhd_4335_mcc/include
 	# CFG 80211 and P2P compile flag
